cypress-tests/cypress/support/e2e.js

```diff
@@ -16,6 +16,7 @@
 // Import commands.js using ES2015 syntax:
 import "cypress-mochawesome-reporter/register";
 import "./commands";
+import "./ucs-commands";
```

Comment:
unless the requests and responses are completely different from the ones that are already available, i'm against creating new set of exchanges and their validations. it just causes maintenance headache and confusion.